### ISSUE TITLE
Suggestion - Show Journal entry

### ISSUE THREAD

**Issue by xlilcasper** [1000nettles/hey-wait#17]
Allow a journal to be opened when Hey Wait! is triggered. This should be shown to any combination of the GM, the triggering player, all players.

**Comment by 1000nettles** [comment]
Thanks for the feature request! Our next steps with any sort of "triggers" will be to trigger a macro which will allow a journal entry to be shown in your macro code. I'll loop back once that's a possibility.

**Comment by 1000nettles** [comment]
I have just released version 0.4.0 of the module which allows you to attach macros to be executed when Hey, Wait! tiles are triggered. Try creating a new macro with the following code and attach it to a tile...

```
JournalEntry.collection.getName("your_journal_entrys_name").show();
```
